Hudl.Mjolnir/Config/MjolnirConfiguration.cs

```diff
@@ -15,54 +17,54 @@ public abstract class MjolnirConfiguration
         /// it through Bulkheads and Circuit Breakers. No timeouts will be applied; a CancellationToken.None will be 
         /// passed to any method that supports cancellation.
         /// </summary>
-        public bool IsEnabled { get; protected set; }
+        public bool IsEnabled { get; set; }
         
         
         /// <summary>
         /// Global Ignore - Timeouts can be globally ignored. Only recommended for use in local/testing environments.
         /// </summary>
-        public bool IgnoreTimeouts { get; protected set; }
+        public bool IgnoreTimeouts { get; set; }
         
         
         /// <summary>
         /// Default Timeouts for commands.
         /// </summary>
-        protected CommandConfiguration DefaultCommandConfiguration;
+        public CommandConfiguration DefaultCommandConfiguration;
         
         
         /// <summary>
         /// Configuring Timeouts for commands.
         /// </summary>
-        protected Dictionary<string, CommandConfiguration> CommandConfigurations;
+        public Dictionary<string, CommandConfiguration> CommandConfigurations;
         
 
        
         /// <summary>
         /// System-wide default. Used if a per-bulkhead config isn't configured.
         /// </summary>
-        protected BulkheadConfiguration DefaultBulkheadConfiguration { get; set; }
+        public BulkheadConfiguration DefaultBulkheadConfiguration { get; set; }
         
         /// <summary>
         /// Per-bulkhead configuration. Key is the argument passed to the Command constructor.
         /// </summary>
-        protected Dictionary<string, BulkheadConfiguration> BulkheadConfigurations { get; set; }
+        public Dictionary<string, BulkheadConfiguration> BulkheadConfigurations { get; set; }
 
       
         
         /// <summary>
         /// Global Enable/Disable - Circuit Breakers can be globally disabled.
         /// </summary>
-        public bool UseCircuitBreakers { get; protected set; }      
+        public bool UseCircuitBreakers { get; set; }      
 
         /// <summary>
         /// System-wide default. Used if a per-breaker config isn't configured.
         /// </summary>
-        protected BreakerConfiguration DefaultBreakerConfiguration { get; set; }
+        public BreakerConfiguration DefaultBreakerConfiguration { get; set; }
         
         /// <summary>
         /// Per-breaker configuration. breaker-key is the argument passed to the Command constructor.
         /// </summary>
-        protected Dictionary<string, BreakerConfiguration> BreakerConfigurations { get; set; }
+        public Dictionary<string, BreakerConfiguration> BreakerConfigurations { get; set; }
```

Comment:
Should the keys for these Dictionaries be `GroupKey`s now that it's the argument to access config?